ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -49,6 +49,75 @@ distro_custom() {
 
     dnf config-manager --disable powertools
 
+    # New Rocky images don't have debuginfo baked into them
+    if [ "$(lsb_release -s -i)" = "Rocky" ]; then
+        # Need to remove the upstream [debuginfo] repos
+        # But need to have the files present so that re-installation is blocked
+        for file in /etc/yum.repos.d/{Rocky-Debuginfo,epel{,{,-testing}-modular}}.repo; do
+            true > $file
```

Comment:
Is this similar to `touch $file`?

Reply:
It would be if the file didn't exist.  But when a file exists `touch` just updates it's _mtime_.  `> $file` creates a file if it doesn't exist and truncates it if it does..  You don't really need the `true` (i.e. just `> $file` is enough) but it placates shellcheck.